### PR TITLE
Moving additional paths into top level of object

### PR DESCRIPTION
**Pull request by dcantatore**
As noted here:
https://github.com/apinf/restivus-swagger/issues/18

### CODE REVIEW

File: server/restivus-swagger.js [hunk 1]
```diff
@@ -85,15 +85,15 @@ Restivus.prototype.addSwagger = function(swaggerPath) {
           doc.definitions = swagger.definitions;
         }
           
-        // Check swagger meta for additional paths
-        if(swagger.meta.paths !== undefined){
-          for (let path in swagger.meta.paths){
+        // Check swagger main object for additional paths
+        if(swagger.paths){
```
Comment (brylie) [inline server/restivus-swagger.js]: Use consistent whitespace. This should be `if (condition) {`

**Review by brylie — CHANGES_REQUESTED** [review]

**Comment by brylie** [comment]
This looks fine. Lets just clean up the whitespace in the file, so that it is consistent. Previously, we had just one developer writing the plugin, but now we have multiple contributors.

**Comment by dcantatore** [comment]
@brylie I have the whitespace corrected file ready for a PR, should I kill this PR and create another?

**Comment by brylie** [comment]
We can still work from this PR, as long as you are making changes to the same branch. Is this PR ready for review?

**Comment by dcantatore** [comment]
Same branch

**Review by brylie — APPROVED** [review]

**Pull request merged**